client/src/components/SelectTabs/EventTypesTab.js

```diff
@@ -25,521 +25,470 @@ import Checkmark from "../../assets/check.png";
-                            alignItems="flex-start"
-                            container
-                            justify="flex-start"
-                            className={classes.inputRow}
-                        >
-                            <Grid xs="2" item>
-                                <InputLabel
-                                    className={classes.descriptionLabel}
-                                >
-                                    Desciption
-                                </InputLabel>
-                            </Grid>
+              <Grid xs="6" item>
+                <Grid
+                  container
+                  direction="row"
+                  alignItems="baseline"
+                  justify="flex-start"
+                  wrap="nowrap"
+                  spacing="2"
+                  className={classes.groupedInput}
+                >
+                  <Grid xs="4" item>
+                    <Input
+                      label="Duration"
+                      name="duration"
+                      type="number"
+                      value={eventBody.duration}
+                      onChange={handleFormChange}
+                      disableUnderline
+                      inputProps={{
+                        min: "1",
+                        style: { textAlign: "center" },
+                      }}
+                      fullWidth
+                    />
+                  </Grid>
+                  <Grid xs="8" item>
+                    <Select
+                      value={unit}
+                      onChange={handleUnitChange}
+                      label="Units"
+                      disableUnderline
+                      fullWidth
+                    >
+                      <MenuItem value={"min"}>minutes</MenuItem>
+                      <MenuItem value={"hour"}>hours</MenuItem>
+                    </Select>
+                  </Grid>
+                </Grid>
+              </Grid>
+            </Grid>
+            <Grid
+              direction="row"
+              alignItems="flex-start"
+              container
+              justify="flex-start"
+              className={classes.inputRow}
+            >
+              <Grid xs="2" item>
+                <InputLabel className={classes.descriptionLabel}>
+                  Desciption
+                </InputLabel>
+              </Grid>
 
-                            <Grid xs="10" className={classes.singleInput} item>
-                                <TextareaAutosize
-                                    rowsMax={6}
-                                    name="description"
-                                    onChange={handleFormChange}
-                                    value={eventBody.description}
-                                    placeholder={`Write a summary and details about your event.
+              <Grid xs="10" className={classes.singleInput} item>
+                <TextareaAutosize
```

Comment:
You can replace this with a `TextField` for a more uniform look and use the `multiline` prop to turn on `TextareaAutosize`! [docs](https://material-ui.com/components/text-fields/#multiline)